endpoints.json

```diff
@@ -80,5 +80,22 @@
       "created_at": "2024-11-26T12:28:45.265Z",
       "votes": 0
     }
+  },
+  "PATCH /api/articles/:article_id/comments": {
+    "description": "returns a given article_id with votes property updated based on request body",
+    "queries": [],
+    "exampleBody": {
+      "inc_votes": 1
+    },
+    "exampleResponse": {
+      "article_id": 2,
+      "title": "Sony Vaio; or, The Laptop",
+      "topic": "mitch",
+      "author": "icellusedkars",
+      "body": "Call me Mitchell. Some years ago—never mind how long precisely—having little or no money in my purse, and nothing particular to interest me on shore, I thought I would buy a laptop about a little and see the codey part of the world. It is a way I have of driving off the spleen and regulating the circulation. Whenever I find myself growing grim about the mouth; whenever it is a damp, drizzly November in my soul; whenever I find myself involuntarily pausing before coffin warehouses, and bringing up the rear of every funeral I meet; and especially whenever my hypos get such an upper hand of me, that it requires a strong moral principle to prevent me from deliberately stepping into the street, and methodically knocking people’s hats off—then, I account it high time to get to coding as soon as I can. This is my substitute for pistol and ball. With a philosophical flourish Cato throws himself upon his sword; I quietly take to the laptop. There is nothing surprising in this. If they but knew it, almost all men in their degree, some time or other, cherish very nearly the same feelings towards the the Vaio with me.",
```

Comment:
You could assert the data type of string to tidy up the test. So long as you assert the id and updated votes explicitly that is fine

Reply:
I've done another pass over these tests to make explicit assertions about the article_id I'm expecting - thanks Mezz! :saluting_face: